languages.go

```diff
@@ -0,0 +1,1208 @@
+package countries
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+type LanguageCode string // string for database/sql/driver.Valuer compatibility
+
+type Language struct {
+	Name string
+	Code LanguageCode
+}
+
+func (LanguageCode) Type() string {
+	return TypeLanguageCode
+}
+
+func (lc LanguageCode) String() string { //nolint:gocyclo
+	switch lc {
+	case LanguageAA:
```

Comment:
I would move all this to a sub package


This way, it's clearer for people who imports

Also, it avoids you to prefix everything related to language with language

It would look like this

```suggestion
package languages

import (
	"encoding/json"
	"fmt"
)

type Code string // string for database/sql/driver.Valuer compatibility

type Language struct {
	Name string
	Code Code
}

func (Code) Type() string {
	return TypeLanguageCode
}

func (lc LanguageCode) String() string { //nolint:gocyclo
	switch lc {
	case CodeAA:
```

Reply:
Whilst that is a fair point, that is not the way this package is written. There are other facets of a country which are all under the `countries` package.

Reply:
You are right. Currencies are aside countries while it's another iso norm